postprocessing/Consumer.py

```diff
@@ -155,7 +116,7 @@ def consume(self, client, frame):
             max_procs_reached = len(self.procList) > self.config.max_procs
             if max_procs_reached:
                 logging.info(
-                    "Maxmimum number of sub-processes reached: %s" % len(self.procList)
+                    "Maxmimum number of sub-processes reached: %s", len(self.procList)
```

Comment:
```suggestion
                    f"Maxmimum number of sub-processes reached: {len(self.procList)}", 
```

Reply:
python logging does lazy evaluation with how I did it